### PR TITLE
Create and add database for book store include role, user, book

### PR DESCRIPTION
**Pull request by hieuneko**

### CODE REVIEW

File: src/main/resources/db/migration/COMMON/V1_0_0_202303021100__create_table_role.sql [hunk 1]
```diff
@@ -0,0 +1,10 @@
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+
+
+CREATE TABLE roles
+(
+    id   UUID         NOT NULL PRIMARY KEY DEFAULT uuid_generate_v4(),
+    name VARCHAR(255) NOT NULL
+);
+
+INSERT INTO roles(name) VALUES ('Admin'), ('Contributor')
```
Comment (voquanghoa) [inline src/main/resources/db/migration/COMMON/V1_0_0_202303021100__create_table_role.sql]: Viết hoa Admin với Contributor

File: src/main/resources/db/migration/COMMON/V1_0_0_202303021101__create_table_user.sql [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    password  VARCHAR(255) NOT NULL,
+    firstName VARCHAR(255),
+    lastName  VARCHAR(255),
+    enabled   BOOLEAN      NOT NULL,
+    avatar    VARCHAR(255),
+    role_id   UUID         NOT NULL,
+    FOREIGN KEY (role_id) REFERENCES roles(id)
+);
+
+INSERT INTO users(username, password, firstname, lastname, enabled, avatar, role_id)
```
Comment (voquanghoa) [inline src/main/resources/db/migration/COMMON/V1_0_0_202303021101__create_table_user.sql]: Viết hoa Admin với Contributor

File: src/main/resources/db/migration/COMMON/V1_0_0_202303021101__create_table_user.sql [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+VALUES ('admin', '123456', 'hieu', 'pham', true, null , (SELECT id FROM roles WHERE name = 'Admin'))
```
Comment (nguyenvanman) [inline src/main/resources/db/migration/COMMON/V1_0_0_202303021101__create_table_user.sql]: Please encrypt password

File: src/main/resources/db/migration/COMMON/V1_0_0_202303021102__create_table_book.sql [hunk 4]
```diff
@@ -0,0 +1,14 @@
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+
+CREATE TABLE books
+(
+    id          UUID         NOT NULL PRIMARY KEY DEFAULT uuid_generate_v4(),
+    title       VARCHAR(255) NOT NULL,
+    author      VARCHAR(255) NOT NULL,
+    description VARCHAR(255),
+    created_at  TIMESTAMP     NOT NULL,
+    updated_at  TIMESTAMP     NOT NULL,
```
Comment (nguyenvanman) [inline src/main/resources/db/migration/COMMON/V1_0_0_202303021102__create_table_book.sql]: this can be nullable

**Review by nguyenvanman — CHANGES_REQUESTED** [review]

File: src/main/resources/db/migration/COMMON/V1_0_0_202303021102__create_table_book.sql (hunk 4 above)
Comment (hieuneko) [reply src/main/resources/db/migration/COMMON/V1_0_0_202303021102__create_table_book.sql]: done

File: src/main/resources/db/migration/COMMON/V1_0_0_202303021102__create_table_book.sql [hunk 5]
```diff
@@ -0,0 +1,14 @@
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+
+CREATE TABLE books
+(
+    id          UUID         NOT NULL PRIMARY KEY DEFAULT uuid_generate_v4(),
+    title       VARCHAR(255) NOT NULL,
+    author      VARCHAR(255) NOT NULL,
+    description VARCHAR(255),
+    created_at  TIMESTAMP     NOT NULL,
```
Comment (voquanghoa) [inline src/main/resources/db/migration/COMMON/V1_0_0_202303021102__create_table_book.sql]: Let default `NOW()`

File: src/main/resources/db/migration/COMMON/V1_0_0_202303021101__create_table_user.sql (hunk 3 above)
Comment (hieuneko) [reply src/main/resources/db/migration/COMMON/V1_0_0_202303021101__create_table_user.sql]: encrypted

File: src/main/resources/db/migration/COMMON/V1_0_0_202303021100__create_table_role.sql (hunk 1 above)
Comment (hieuneko) [reply src/main/resources/db/migration/COMMON/V1_0_0_202303021100__create_table_role.sql]: done

File: src/main/resources/db/migration/COMMON/V1_0_0_202303021101__create_table_user.sql (hunk 2 above)
Comment (hieuneko) [reply src/main/resources/db/migration/COMMON/V1_0_0_202303021101__create_table_user.sql]: done

File: src/main/resources/db/migration/COMMON/V1_0_0_202303021102__create_table_book.sql (hunk 5 above)
Comment (hieuneko) [reply src/main/resources/db/migration/COMMON/V1_0_0_202303021102__create_table_book.sql]: done

**Pull request merged**